lib/dry/rails/features/safe_params.rb

```diff
@@ -68,7 +70,11 @@ def schemas
 
         # @api private
         def set_safe_params
-          @safe_params = schemas[action_name.to_sym].(request.params)
+          @safe_params ||= {}
+
+          schemas.each do |action, schema|
+            @safe_params[action] = schema.(request.params)
+          end
```

Comment:
@gotar hmm why does it have to go through all defined schemas instead of just fetching a schema for the current action and setting `@safe_params`?

Reply:
When I tested, going just through defined do not work for me. Was skipping other then the last one. That was only solution that worked for me. But maybe you will find a better one? Going through all do not affect it anyhow, so I left it in that way.

Reply:
@gotar it works when you just do:

```ruby
schema = schemas[action_name.to_sym]
return unless schema
@safe_params = schema.(request.params)
```

+ removig the fetch part from `safe_params` reader.

I'll merge this in if you could tweak it like that.